projects/core/src/occ/config/occ-config.ts

```diff
@@ -13,6 +13,7 @@ export abstract class OccConfig extends ServerConfig {
       endpoints?: {
         [endpoint: string]: string;
       };
+      userIdentifier?: string;
```

Comment:
I was thinking, maybe this could be a type or an enum instead of a string. That way when you write the config in the app module you get some nice code completion. It's something we do in the IconConfig and SiteContextConfig Either like:
`export type UserIdentifier = 'uid' | 'customerId';`

or
```
export enum USER_IDETIFIERS {
  UID = 'uid',
  CUSTOMER_ID = 'customerId',
}
```

Reply:
Does it require to make additional imports in the app.module.ts to override the config?  Or can we still just provide a string value?

If not, I agree this might be a useful improvement.

Reply:
The first option will not require an addition import in the app.module. When you enter the value it will look just like a string but there will be code completion.

Reply:
First option it is then!